pom.xml

```diff
@@ -22,7 +22,7 @@
         <org.mapstruct.version>1.3.0.Final</org.mapstruct.version>
         <spring-boot-dependencies.version>2.1.15.RELEASE</spring-boot-dependencies.version>
         <spring-boot-maven-plugin.version>2.1.15.RELEASE</spring-boot-maven-plugin.version>
-        <start-class>uk.gov.companieshouse.chdorderconsumer.ChdOrderConsumerApplication</start-class>
+        <start-class>uk.gov.companieshouse.chdorderconsumer.ChdOrderApiApplication</start-class>
```

Comment:
should chdorderconsumer be chdorderapi? Also which branch are we supposed to merging into, master or main?

Reply:
it should be chd.order.api

Reply:
And we need to merge to master for the pipeline to work